docs/content/what-is-the-temporal-server.md

```diff
@@ -0,0 +1,81 @@
+---
+id: what-is-the-temporal-server
+title: What is the Temporal Server?
+description: The Temporal Server is a scalable multi-tenant entity made up of individually scalable services, that is capable of handling millions to billions of Workflows Executions concurrently.
+---
+
+import CenteredImage from "../components/CenteredImage.js"
+import RelatedReadList from '../components/RelatedReadList.js'
+
+The Temporal Server is a highly scalable multi-tenant system that is capable of tracking the state of millions to billions of Workflows Executions concurrently.
+
+An instance of the Temporal Server is a group of individually scalable services.
+These processes can run independently or be grouped together into shared processes on one or more physical or virtual machines.
+For live (production) environments we recommend that each service is running independently, as they have different scale out requirements and troubleshooting becomes easier.
+The Temporal Server employs various sharding techniques to ensure internal scalability.
+An instance of the Server is often referred to as a Cluster.
+
+<!-- TODO <RelatedReadList
+readlist={[
+["What is Multi-Cluster Replication", "/docs/content/what-is-multi-cluster-replication"]
+]}
+/> -->
+
+<CenteredImage
+imagePath="/diagrams/temporal-system-entity-relationship.svg"
+imageSize="100"
+title="The Temporal Server topology"
+/>
+
+### Frontend Service
+
+The Frontend service exposes a strongly typed [Proto API](https://github.com/temporalio/api/blob/master/temporal/api/workflowservice/v1/service.proto).
+
+The Frontend service is responsible for all in-bound calls, including [Multi-cluster Replication](/docs/server/multi-cluster) related calls that are invoked by a remote cluster.
+
```

Comment:
```suggestion
The Frontend service is a singleton (not horizontally scalable) service responsible for rate limiting, authorizing, validating and routing all in-bound calls:
	- Domain CRUD
	- External Events
	- Poll APIs for workers
	- Visibility API requests
	- Admin CLI operations 
	- including [Multi-cluster Replication](/docs/server/multi-cluster) related calls that are invoked by a remote cluster.

Every inbound request has a workflow ID, which is hashed for routing by the Frontend service.
The Frontend service has visibility into the hash rings maintains membership info for routing, having knowledge of how many nodes are in the cluster.

Rate limiting is applied per host and per namespace.
```

todo: i think rate limiting and authorization info is pretty important - should we flesh this out?